### PR TITLE
update swift version

### PR DESCRIPTION
**Pull request by odariusgeorge**
Xcode 10.2 - carthage - unsupported swift version (3.0 not supported anymore)

### CODE REVIEW

**Comment by acrabb** [comment]
eta on this?

**Comment by shayneoneill** [comment]
Bump. This is a massive code-breaker

**Comment by danielgratwickeaccedo** [comment]
Keen to find out any available ETAs on this.

**Comment by dmzza** [comment]
I hope this will be merged soon. But in case anyone wasn't aware of this trick...

While we wait for this to be merged, you can always **change your podfile** to this:

```ruby
pod 'SwiftSocket', :git => 'https://github.com/odariusgeorge/SwiftSocket.git', :branch => 'patch-1'
```

It works for me. 👍

**Comment by shayneoneill** [comment]
If this will take time to integrate it, could I suggest at least updating the documentation to reflect the interim solution (with a note saying it would be prudent to revert to the non versioned Podspec when the patch is merged and published). Its a pretty disruptive bug

**Comment by AZOM** [comment]
I would really appreciate if this would be fixed. Why not using Swift 5 directly instead of 4.2?

**Comment by eschos24** [comment]
I made an updated PR for Swift 5 (#186) with a couple additional (very small) fixes.

**Comment by kveldkamp** [comment]
Please merge in this PR or #186, I also had the same issue in our production app. Thanks @dmzza for the suggestion, worked perfectly

File: SwiftSocket.podspec [hunk 1]
```diff
@@ -23,6 +23,6 @@ Pod::Spec.new do |s|
     :tag => s.version
   }
   s.source_files  = 'Sources/**/*.{swift,c}'
-  s.pod_target_xcconfig = { 'SWIFT_VERSION' => '3' }
+  s.pod_target_xcconfig = { 'SWIFT_VERSION' => '4.2' }
```
Comment (purifier1990) [inline SwiftSocket.podspec]: Should we try using swift5 instead of 4.2?

**Pull request merged**